internal/extsvc/bitbucketserver/client.go

```
@@ -444,7 +450,13 @@ func (c *Client) CreatePullRequest(ctx context.Context, pr *PullRequest) error {
 	err := c.send(ctx, "POST", path, nil, payload, pr)
 	if err != nil {
 		if IsDuplicatePullRequest(err) {
-			return ErrAlreadyExists
+			pr, extractErr := ExtractDuplicatePullRequest(err)
+			if extractErr != nil {
+				log15.Error("Extracting existsing PR", "err", extractErr)
```

I think we should return an error here rather than logging.